libass/ass_parse.c

```diff
@@ -741,6 +741,16 @@ char *parse_tags(ASS_Renderer *render_priv, char *p, char *end, double pwr,
                 render_priv->state.be = val;
             } else
                 render_priv->state.be = 0;
+        } else if (complex_tag("box")) {
+            if (nargs == 4) {
+                render_priv->state.margin_x0 = argtoi(args[0]);
+                render_priv->state.margin_y0 = argtoi(args[1]);
+                render_priv->state.margin_x1 =
+                    render_priv->track->PlayResX - argtoi(args[2]);
+                render_priv->state.margin_y1 =
+                    render_priv->track->PlayResY - argtoi(args[3]);
```

Comment:
Does this work correctly when libass is told to draw subtitles in black bars around the video?